src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_validate.c

```diff
@@ -50,7 +50,7 @@ static int wm_agent_upgrade_validate_system(const char *platform, const char *os
  * @retval WM_UPGRADE_URL_NOT_FOUND
  * @retval WM_UPGRADE_WPK_VERSION_DOES_NOT_EXIST
  * */
-static int wm_agent_upgrade_validate_wpk_version(const wm_agent_info *agent_info, wm_upgrade_task *task, char *wpk_version);
+static int wm_agent_upgrade_validate_wpk_version(const wm_agent_info *agent_info, wm_upgrade_task *task, char *wpk_version, char *wpk_repository_config);
```

Comment:
Add the new parameter to the description

Reply:
`wpk_repository_config` should be `const`